examples/client/cli-change.txt

```diff
@@ -0,0 +1,1918 @@
+diff --git a/examples/client/CMakeLists.txt b/examples/client/CMakeLists.txt
```

Comment:
This looks like a leftover file you inadvertently committed :wink:

Reply:
Sorry.  It's now deleted.